src/concrete_intersection.jl

```diff
@@ -154,5 +154,35 @@ function intersection(P1::Union{HPolytope{N}, VPolytope{N}},
     return output_type(Pint)
 end
 
+"""
+    intersection(P1::S1, P2::S2) where {S1<:AbstractPolytope{N},
+                                        S2<:AbstractPolytope{N}} where N
+
+Compute the intersection of two polytopic sets.
+
+### Input
+
+- `P1`          -- polytope
+- `P2`          -- another polytope
+
+### Output
+
+The polytope obtained by the intersection of `P1` and `P2`, in V-representation.
```

Comment:
shall we add something in the lines of: "This is a fallback implementation that requires `Polyhedra` to evaluate the concrete intersection. In the general case, the argument sets are converted to `VPolytope` through the `vertices_list` function."